File: spray-calculator/WebApi/Agrovision.Spray_Calculator.Api/Models/SprayModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Agrovision.Spray_Calculator.Api.Models
{
    public class SprayModel
    {
        public long Id { get; set; }
        public Guid SprayerKey { get; set; }
        public string Description { get; set; }
        public double SparyerVolumeL { get; set; }
        public bool IsActive { get; set; }
    }
}

File: spray-calculator/WebApp/Agrovision/src/app/models/spray/spray.model.ts
export class SprayModel {

    constructor(description: string, sparyerVolumeL: number) {
        this.description = description;
        this.sparyerVolumeL = sparyerVolumeL;

    }
    id: bigint;
    sprayerKey: string;
    description: string;
    sparyerVolumeL: number;
    isActive: boolean;
}
